piksi_multi_cpp/src/sbp_callback_handler/sbp_callback_handler_relay/ros_time_handler.cc

```diff
@@ -0,0 +1,118 @@
+#include "piksi_multi_cpp/sbp_callback_handler/sbp_callback_handler_relay/ros_time_handler.h"
+
+#include <libsbp_ros_msgs/ros_conversion.h>
+#include <ros/node_handle.h>
+#include <cmath>
+#include <functional>
+
+namespace piksi_multi_cpp {
+namespace s = std::placeholders;
+namespace lrm = libsbp_ros_msgs;
+
+RosTimeHandler::RosTimeHandler(const std::shared_ptr<sbp_state_t>& state)
+    : gps_time_handler_{std::bind(&RosTimeHandler::callbackToGpsTime, this,
+                                  s::_1),
+                        SBP_MSG_GPS_TIME, state},
+      utc_time_handler_{
+          std::bind(&RosTimeHandler::callbackToUtcTime, this, s::_1),
+          SBP_MSG_UTC_TIME, state} {
+  // Check if user wants to stamp data with GPS time.
+  ros::NodeHandle nh_node("~");
+  // TODO(rikba): Instead of setting a member variable it would be nicer to have
+  // the lambda callbacks optional.
+  nh_node.getParam("use_gps_time", use_gps_time_);
```

Comment:
Hm, thats one of the config things i would set in a setter/constructor, to make it more verbose

Reply:
I tried, but that you need you need to carry the flag all the way through the code. I added the request to https://github.com/ethz-asl/ethz_piksi_ros/issues/102 and we should have a consistent param reading before release.